Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -240,7 +240,8 @@ void GlobalObject::initialize_global_object()
     define_direct_property(vm.names.console, heap().allocate<ConsoleObject>(*this, *this), attr);
     define_direct_property(vm.names.Atomics, heap().allocate<AtomicsObject>(*this, *this), attr);
     define_direct_property(vm.names.Math, heap().allocate<MathObject>(*this, *this), attr);
-    define_direct_property(vm.names.JSON, heap().allocate<JSONObject>(*this, *this), attr);
+    m_json_object = heap().allocate<JSONObject>(*this, *this);
```

Comment:
no need for this. Just `get_without_side_effects` the parse function at the end of `initialize_global_object`, and don't forget to visit it :)

Reply:
Ooh that's much nicer